app/assets/javascripts/views/column_visibility_button_view.js

```diff
@@ -20,19 +20,19 @@ module.exports = Backbone.View.extend({
     var icon = "";
     switch(this.options.columnView.id) {
     case 'done' :
-      icon = '<i class="mi md-18">done</i> ';
+      icon = '<i class="mi md-18 mi--sidebar">done</i> ';
       break;
     case 'in_progress' :
-      icon = '<i class="mi md-18">inbox</i> ';
+      icon = '<i class="mi md-18 mi--sidebar">inbox</i> ';
       break;
     case 'backlog' :
-      icon = '<i class="mi md-18">list</i> ';
+      icon = '<i class="mi md-18 mi--sidebar">list</i> ';
```

Comment:
mi is used by the [material_icons gem](https://github.com/Angelmmiguel/material_icons). I've used mi--sidebar to signalize it is used only at the sidebar.